### DISCUSSION TITLE
Does pivot point shares for each layer or item?

### DISCUSSION THREAD

**Discussion by jaxjixmix** [keiwando/toonsquid#449] [General]
I have an issue where I need to change the pivot point because it affects other elements. 

I have already separated each element into layers, but they are still sharing the same pivot point.

https://github.com/keiwando/toonsquid/assets/702899/a1d4a062-f0e0-4d3c-84f7-946d7f2570f1

**Comment by keiwando** [comment]
That's the [temporary pivot](https://toonsquid.com/handbook/settings/tools/#temporary-pivots-enabled) for when you are editing either the path control points of a path or vector layer or the pixels within a pixel layer. It's not the layer's pivot property. You must have previously enabled that either in the settings or by tapping on the selected layer and selecting "Enable pivot".
The transform tool in your video currently has the "edit path" option enabled at the top. If you disable that, the transform tool will instead edit the layer transform properties and show you the actual persistent pivot of your layer.

**Reply by SirTopHat997** [reply]
This was my issue exactly, I really appreciate the help
